index.html

```diff
@@ -1,9 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Saira:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link href="css/headerFooter.css" rel="stylesheet" />
+    <link href="css/aboutUs.css" rel="stylesheet" />
+
     <meta charset="UTF-8" />
```

Comment:
Meta tags are before style links in the hierarchy

Reply:
fixed